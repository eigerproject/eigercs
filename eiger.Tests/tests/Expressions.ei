~ Basic Arithmetic
emitln(5 + 3 * 2)
emitln(10 / 2 - 1)
emitln(7 * (5 - 3))

~ String concatenation
emitln("Hello " + "world!")