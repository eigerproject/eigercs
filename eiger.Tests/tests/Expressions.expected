11
4
14
Hello world!